bundle/src/main/resources/CQ-INF/nodetypes/cqsm_nodetypes.cnd

```diff
@@ -27,5 +27,6 @@
   - cqsm:executionMode (string)
   - cqsm:executionSchedule (date)
   - cqsm:executionLast (date)
+  - cqsm:dryRunLast (date)

```

Comment:
will apm be still backward compatible? I think that migration will be extremely easy

change cqsm to apm in properties and file names... so that I'd like to encourage you to clean it up and use 'apm' name everywhere;

if needed we could provide *.cqsm to *.apm conversion tool which will be also extremely easy and could be built in into new APM (new button Upload Legacy Script or sth like that)

Reply:
Ok, I think, we could do that in selected pull request.